SDWebImage/Core/UIView+WebCacheOperation.m

```diff
@@ -60,7 +60,7 @@ - (void)sd_cancelImageLoadOperationWithKey:(nullable NSString *)key {
             operation = [operationDictionary objectForKey:key];
         }
         if (operation) {
-            if ([operation conformsToProtocol:@protocol(SDWebImageOperation)]) {
```

Comment:
From [this doc](https://developer.apple.com/documentation/objectivec/nsobject/1418893-conformstoprotocol):

<img width="925" alt="Screen Shot 2022-07-28 at 3 28 05 PM" src="https://user-images.githubusercontent.com/522951/181647790-0d3b665d-aa1c-46b9-9f4a-9d5e5014d75c.png">